tiatoolbox/wsicore/wsireader.py

```diff
@@ -1089,35 +1275,54 @@ class VirtualWSIReader(WSIReader):
     - .png
     - :class:`numpy.ndarray`
 
+    This reader uses :func:`tiatoolbox.utils.image.sub_pixel_read` to
+    allow reading low resolution images as if they are larger i.e. with
+    'virtual' pyramid resolutions. This is useful for reading low
+    resolution masks as if they were stretched to overlay a higher
+    resolution WSI.
+
+    Extra key-word arugments given to :func:`~WSIReader.read_region`
+    and :func:`~WSIReader.read_bounds` will be passed to
+    :func:`~tiatoolbox.utils.image.sub_pixel_read`.
+
     Attributes:
         img (:class:`numpy.ndarray`)
+        mode (str)
 
     Args:
-        input_img (str or pathlib.Path or :class:`numpy.ndarray`): input path to WSI.
-
+        input_img (str, pathlib.Path, ndarray): input path to WSI.
+        info (WSIMeta): Metadata for the virtual wsi.
+        mode (str): Mode of the input image. Default is 'rgb'. Allowed
+            values are: rgb, bool.
     """
```

Comment:
Please add a new line at the end.

Reply:
Fixed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/61/commits/5b0e0ecc1fcb9ce989d907483bef9a2b0b2c7aa5